### PR TITLE
Make route-templates typeable

### PR DESCRIPTION
**Pull request by istarkov**
## Description

Now route-templates is normal ts file. And can be typechecked

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: fixtures/webstudio-remix-vercel/app/constants.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const assetBaseUrl = "/assets/";
+export const imageBaseUrl = "/assets/";
```
Comment (TrySound) [inline fixtures/webstudio-remix-vercel/app/constants.ts:2]: Aren't they generated?

**Review by TrySound — APPROVED** [review]

File: fixtures/webstudio-remix-vercel/app/constants.ts (hunk 1 above)
Comment (JayaKrishnaNamburu) [inline fixtures/webstudio-remix-vercel/app/constants.ts:2]: Should we leave a comment, that this should match with this step in prebuild.
https://github.com/webstudio-is/webstudio/blob/main/packages/cli/src/config.ts#L14

Comment (istarkov) [reply fixtures/webstudio-remix-vercel/app/constants.ts:2]: image

Im gonna remove that step in next iterations.

Comment (JayaKrishnaNamburu) [reply fixtures/webstudio-remix-vercel/app/constants.ts:2]: cool 👍

Comment (istarkov) [reply fixtures/webstudio-remix-vercel/app/constants.ts:2]: As of now found the only way is to rename constants.ts to constants.mjs and eval using node 
```
node --experimental-specifier-resolution=node --input-type=module --eval "import * as consts from './app/constants.mjs'; console.log(JSON.stringify(consts))"
```

**Pull request merged**